docker-compose.yaml

```diff
@@ -10,3 +10,23 @@ services:
       POSTGRES_USER: "$DB_USER"
       POSTGRES_PASSWORD: "$DB_PASSWORD"
       POSTGRES_DB: "$DB_NAME"
+  api:
+    restart: always
+    build: 
+      context: .
+      dockerfile: src/backend/Dockerfile
```

Comment:
Am getting `failed to initialize database, got error failed to connect to `host=host.docker.internal user=root database=boardly`: hostname resolving error (lookup host.docker.internal on 127.0.0.11:53: no such host)`  and my guess is the `DB_HOST` in the `.env`